codebuild/release/prod-release.yml

```diff
@@ -0,0 +1,28 @@
+version: 0.2
+
+env:
+  variables:
+    BRANCH: "master"
+  secrets-manager:
+    TWINE_USERNAME: PyPiAdmin:username 
+    TWINE_PASSWORD: PyPiAdmin:password
```

Comment:
This is nice! It's too bad this feature doesn't seem to support downloading binary secrets (AFAICT).